src/components/table/table-meta-header/TableMetaHeader.tsx

```
@@ -48,16 +48,42 @@ export default function TableMetaHeader({
+        const filterDefaultValue = filterConfig.initialFilters?.[filter.key] as string
+        if (filterDefaultValue) {
+          defaultValues = isMulti ? filterDefaultValue.split(',') : filterDefaultValue
+        }
+        return {
+          key: filter.key,
+          values: defaultValues,
+          type: filter.type,
+        }
+      }
+
+      setDefaultFilters([...filters?.map(mapFn)])
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)